index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
         <img src="#" alt="FontIcon" class="font-icon" />
         <img src="#" alt="OtherImg" class="fourth-illustration" />
       </figure>
-      <section class="product-container">
-        <ul class="product-list">
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
+    </main>
+    <section class="product-container">
+      <ul class="product-list">
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+      </ul>
+    </section>
+    <section class="spotlight-container">
+      <div class="spotlight-header">
+        <h2 class="spotlight-heading">Spotlight Digital Assets Selections</h2>
+        <a href="#" class="explore-featured">Explore Featured Works &gt;</a>
+      </div>
+      <div class="spotlight-product-container">
+        <div class="spotlight-product">
+          <img src="#" alt="SpotlightProduct" />
+          <div class="spotlight-product-details">
+            <a class="spotlight-product-title">Flexsoul</a>
+            <p class="spotlight-price">$12.00</p>
+            <ul class="spotlight-tags">
```

Comment:
It's not a list.
A list is an enumeration of homogeneous elements with the same meaning and styling. This means that any number of items can be added to the list, and they should all look the same.

Reply:
Im using the list only for the tags so the styling is the same for them